source/mainnet/smart-contracts/tutorials/sponsoredTransactions/index.rst

```diff
@@ -0,0 +1,47 @@
+.. include:: ../../../variables.rst
+.. _sponsoredTransactions:
+
+.. image:: sponsoredTransactions.png
+   :width: 40%
+   :align: center
+   :alt: An eSealing dApp
+
+=============================
+A Sponsored Transactions dApp
+=============================
+
+In this tutorial, you are going to get familiar with sponsored transactions,
+how to send a transaction from a back-end server to the blockchain, and how to connect your Concordium mobile wallet via walletConnect to the front-end.
```

Comment:
Replace "Concordium mobile wallet" with |mw-gen2| since it is only the newer mobile wallets that support WalletConnect.